spinta/api/validators.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, Any
+
+from pydantic import BaseModel, Field, ConfigDict
+
+
+class ClientAddData(BaseModel):
+    client_name: Optional[str] = None
```

Comment:
In other places the pipes have been used to define types, maybe let's follow that here as well?

```suggestion
    client_name: str | None = None
```

Reply:
Oh, that's a bummer, I thought improting annotations from future would suffice, ignore then 😑